### PR TITLE
Update task.json creation instructions

### PR DESCRIPTION
**Pull request by Aman-Pandey-afk**
The PR aims to fix the problem that there is no ''Create tasks.json file from template'' option shown and I had to check stack overflow to conclude that just pressing enter after configure task will create a task.json file. Also, just pressing F1 instead of ctrl+shift+p launches the configure task option.

### CODE REVIEW

**Comment by Calinou** [comment]
My experience is that VS Code will not _always_ create a `tasks.json` file from a template if you have extensions that supply tasks.

For instance, this is what I see by default in a Node.js project:

image

**Comment by raulsntos** [comment]
Also, the option will not show up if the `tasks.json` file already exists. See Microsoft's documentation: https://code.visualstudio.com/docs/editor/tasks#_custom-tasks

**Comment by Aman-Pandey-afk** [comment]
> Also, the option will not show up if the `tasks.json` file already exists. See Microsoft's documentation: https://code.visualstudio.com/docs/editor/tasks#_custom-tasks

I was setting up for the first time and didn't get that option. I guess we can simply change any of the templates, so I thought doc should be updated to reflect this. (Just select what label is available by pressing enter and modify it accordingly)

**Comment by raulsntos** [comment]
> I was setting up for the first time
Even if the current workspace doesn't have a `tasks.json` file, the option also doesn't show up if there is a user `tasks.json` (in Linux the path is `$HOME/.config/Code/User/tasks.json`).

**Comment by Aman-Pandey-afk** [comment]
Yeah, so doc requires clarity regarding task configuration in my opinion as I stumbled upon this thing (although a quick search is all it takes). Leave it to you guys to consider or not.

**Comment by pycbouh** [comment]
> Yeah, so doc requires clarity regarding task configuration in my opinion as one could stumble upon this thing (although a quick search is all it takes). Leave it to you guys to consider or not :)

Feel free to edit this PR to mention both possibilities.

**Comment by Aman-Pandey-afk** [comment]
Rebased the changes, now it reflects both possibilities, any other suggestion will be helpful.

File: development/cpp/configuring_an_ide/visual_studio_code.rst [hunk 1]
```diff
@@ -27,6 +27,8 @@ Importing the project
 
 .. figure:: img/vscode_create_tasksjson_others.png
    :align: center
+ 
+- It can happen that no such option as **Create tasks.json file from template** is available. In that case, select any of the tasks available and press Enter
```
[the remark this answers is not in the document]
Comment (Aman-Pandey-afk) [reply development/cpp/configuring_an_ide/visual_studio_code.rst:31]: Yeah I kind of thought that a link will be better, a user task.json is not correct. Also, thanks for whitespace suggestion. So should I add the link and required instructions and rebase the commit?

[the remark this answers is not in the document]
Comment (raulsntos) [reply development/cpp/configuring_an_ide/visual_studio_code.rst:31]: Yeah, I think it'd be better so if you agree go ahead.

[the remark this answers is not in the document]
Comment (Aman-Pandey-afk) [reply development/cpp/configuring_an_ide/visual_studio_code.rst]: Committed the changes, I hope it's better now.

[the remark this answers is not in the document]
Comment (Aman-Pandey-afk) [reply development/cpp/configuring_an_ide/visual_studio_code.rst]: Have added the required changes, can you review the pull request @raulsntos

**Review by raulsntos — APPROVED** [review]
LGTM.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you and congrats on your first contribution to Godot's docs! 🎉 
Also thanks to the reviewers :)

**Pull request merged**